examples/enforce_coordinator.rs

```diff
@@ -14,24 +16,28 @@ use scylla::statement::prepared::PreparedStatement;
 /// Checks whether the result matches the expected values (i.e. ones stored in peers metadata).
 async fn query_system_local_and_verify(
     session: &Session,
-    node: &Arc<Node>,
+    enforced_node: &Arc<Node>,
     query_local: &PreparedStatement,
 ) {
-    let (actual_host_id, actual_node_ip) = session
+    let result = session
         .execute_unpaged(query_local, ())
         .await
         .unwrap()
         .into_rows_result()
-        .unwrap()
-        .single_row::<(uuid::Uuid, IpAddr)>()
         .unwrap();
 
+    let (actual_host_id, actual_node_ip) = result.single_row::<(uuid::Uuid, IpAddr)>().unwrap();
     println!(
         "queried host_id: {}; queried node_ip: {}",
         actual_host_id, actual_node_ip
     );
-    assert_eq!(node.host_id, actual_host_id);
-    assert_eq!(node.address.ip(), actual_node_ip);
+
+    assert_eq!(enforced_node.host_id, actual_host_id);
+    assert_eq!(enforced_node.address.ip(), actual_node_ip);
+
+    let coordinator_node = result.request_coordinator().node();
+    assert_eq!(coordinator_node.host_id, actual_host_id);
+    assert_eq!(coordinator_node.address.ip(), actual_node_ip);
 }
```

Comment:
I think this fragment is not noticeable enough.
Can you write a comment explaining what is the difference between `actual_host/node` (system.local queries return information about the queried node) and `coordinator_node` (which works driver-side and is a public API).

Comment on `query_system_local_and_verify` should also be updated - now the function checks more stuff.